lms/djangoapps/teams/tests/test_serializers.py

```diff
@@ -31,7 +36,36 @@ def setUp(self):
         )
 
 
-class BaseTopicSerializerTestCase(TopicTestCase):
+class MembershipSerializerTestCase(SerializerTestCase):
+    """
+    Tests for the membership serializer.
+    """
+
+    def setUp(self):
+        super(MembershipSerializerTestCase, self).setUp()
+        self.team = CourseTeamFactory.create(
+            course_id=self.course.id,
+            topic_id=self.course.teams_topics[0]['id']
+        )
+        self.user = UserFactory.create()
+        CourseEnrollmentFactory.create(user=self.user, course_id=self.course.id)
+        self.team_membership = CourseTeamMembershipFactory.create(team=self.team, user=self.user)
+
+    def test_membership_serializer_expand_user_and_team(self):
+        """Verify that the membership serializer only expands the user and
```

Comment:
Nit: the first sentence of a docstring should be a one line summary:

https://github.com/edx/edx-platform/wiki/Python-Guidelines